README.adoc

```diff
@@ -267,45 +266,46 @@ The methods that you wrote in the [hotspot=Consumer file-0]`Consumer` class coul
 party service, you should separate your `GET`/`POST` requests from your data consumption.
 
 // =================================================================================================
-// Building and running the application
+// Running the application
 // =================================================================================================
 
-[role=command]
-include::{common-includes}/mvnbuild.adoc[]
+== Running the application
 
-When the server is running, you can find your service at http://localhost:9080/artists[http://localhost:9080/artists^]
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-[role=command]
-include::{common-includes}/mvncompile.adoc[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
+
+You can now access the endpoints at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists,
+and at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. 
 
 // =================================================================================================
 // Testing deserialization
 // =================================================================================================
 
 == Testing deserialization
-ConsumingRestTest.java
+ConsumingRestIT.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestIT.java[]
 ----
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `ConsumingRestTest` class.#
-`src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java` 
+#Create the `ConsumingRestIT` class.#
+`src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestIT.java` 
 ----
 
-Maven finds and executes all tests under `it/` and each test method must be marked with the [hotspot=test-1 hotspot=test-2 hotspot=test-3 hotspot=test-4]`@Test` annotation.
+Maven finds and executes all tests under `src/test/java/it/` and each test method must be marked with the [hotspot=test-1 hotspot=test-2 hotspot=test-3 hotspot=test-4]`@Test` annotation.
 
-You can use the [hotspot=BeforeClass]`@BeforeClass` and `@AfterClass` annotations to perform any one time setup and teardown
-tasks before and after all of your tests execute, as well as the [hotspot=Before]`@Before` and [hotspot=After]`@After` annotations
+You can use the [hotspot=BeforeAll]`@BeforeAll` and `@AfterAll` annotations to perform any one time setup and teardown
+tasks before and after all of your tests execute, as well as the [hotspot=BeforeEach]`@BeforeEach` and [hotspot=AfterEach]`@AfterEach` annotations
```

Comment:
Change
"You can use the [hotspot=BeforeAll]`@BeforeAll` and `@AfterAll` annotations to perform any one time setup and teardown tasks before and after all of your tests execute, as well as the [hotspot=BeforeEach]`@BeforeEach` and [hotspot=AfterEach]`@AfterEach` annotations to do the same but for each individual test case."
To
"You can use the [hotspot=BeforeAll]`@BeforeAll` and `@AfterAll` annotations to perform any one-time setup and teardown tasks before and after all of your tests run. You can also use the [hotspot=BeforeEach]`@BeforeEach` and [hotspot=AfterEach]`@AfterEach` annotations to perform setup and teardown tasks for individual test cases."